src/MongoDB.AspNetCore.OData/MongoEnableQueryAttribute.cs

```diff
@@ -58,6 +61,11 @@ public override IQueryable ApplyQuery(IQueryable queryable, ODataQueryOptions qu
 
         var ignoreQueryOptions = AllowedQueryOptions.Select | AllowedQueryOptions.Expand;
         queryable = queryOptions.ApplyTo(queryable, ignoreQueryOptions);
+
+        // Expression generated by the oData library for substring is not translatable by Mongo LINQ Provider
```

Comment:
Do not need the comment here. It would be more useful to have it on the `RewriteExpression` method.

Reply:
Done.